### PR TITLE
[FEATURE] implement firstCategory property

### PR DESCRIPTION
**Pull request by h3nn3s**
With having a firstCategory property, it would easyly possible
to implement a grouping by category in Fluid -
e.g. for a category based accordeon.

Resolves: #382

### CODE REVIEW

File: Classes/Domain/Model/Address.php [hunk 1]
```diff
@@ -609,4 +609,20 @@ public function getFullName(): string
         ];
         return implode(' ', array_filter($list));
     }
+
+    /**
+     * Get first category
+     *
+     * @return \TYPO3\CMS\Extbase\Domain\Model\Category
+     */
+    public function getFirstCategory()
```
Comment (georgringer) [inline Classes/Domain/Model/Address.php:618]: please add return type + also nullable

File: Classes/Domain/Model/Address.php [hunk 2]
```diff
@@ -609,4 +609,20 @@ public function getFullName(): string
         ];
         return implode(' ', array_filter($list));
     }
+
+    /**
+     * Get first category
+     *
+     * @return \TYPO3\CMS\Extbase\Domain\Model\Category
```
Comment (georgringer) [inline Classes/Domain/Model/Address.php:616]: also null is possible

File: Documentation/Tutorials/Templates/GroupAddressRecords/Index.rst [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+
+The following example will group all given address records by the property "firstCategory".
+
+.. code-block:: html
+
+	<f:if condition="{addresses}">
+		<f:then>
+			<div style="border:1px solid red">
+				<f:groupedFor each="{addresses}" as="groupedAddresses" groupBy="firstCategory" groupKey="cat">
```
Comment (georgringer) [inline Documentation/Tutorials/Templates/GroupAddressRecords/Index.rst:23]: change cat to category, no need to save chars

**Review by georgringer — CHANGES_REQUESTED** [review]

**Comment by h3nn3s** [comment]
Thanks for reviewing, Georg

**Review by georgringer — APPROVED** [review]

**Pull request merged**